site/docs/red-team/plugins/rag-full-document-exfiltration.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_label: RAG Full Document Exfiltration
+---
+
+# RAG Full Document Exfiltration Plugin
+
+The RAG Full Document Exfiltration plugin is designed to identify vulnerabilities where an AI model might inadvertently expose entire documents during retrieval-augmented generation processes.
```

Comment:
Give more context here. RAGs often contain internal documents but we don't want to expose them.